src/renderer/src/scenes/dir-select-scene/DirSelectScene.tsx

```diff
@@ -0,0 +1,65 @@
+import { createSignal } from "solid-js";
+import "./styles.css";
+import Button from "@renderer/components/button/Button";
+
+export default function DirSelectScene() {
+  const [dir, setDir] = createSignal("");
+
+  const selectDir = async () => {
+    const opt = await window.api.request("dir::select");
+    if (opt.isNone) {
+      return;
+    }
+
+    setDir(opt.value);
+  };
+
+  const autodetectDir = async () => {
```

Comment:
The function shall be run `onMount` considering that for most cases it will most likely catch the osu directory.

Reply:
Added it!